zeebe/dynamic-config/src/main/java/io/camunda/zeebe/dynamic/config/changes/PartitionChangeExecutor.java

```diff
@@ -46,6 +46,19 @@ ActorFuture<Void> join(
    */
   ActorFuture<Void> leave(int partitionId);
 
+  /**
+   * The implementation of this method must bootstrap the partition with a single replica. The
```

Comment:
:thought_balloon: We could also document here that the partition config is copied from partition 1. (Or leave the specific partition unspecified, just say that we copy some partition config).